packages/services/f-feature-management-vue3/vite.config.js

```diff
@@ -0,0 +1,11 @@
+import path from 'path';
+
+export default {
+    build: {
+        target: 'es2015',
+        lib: {
+            entry: path.resolve(__dirname, 'src/index.js'),
+            name: 'f-feature-management-vue3'
+        }
+    }
```

Comment:
I _think_ if you want to exclude your `peerDependencies` from the final bundle you'll need to add some additional config here, [like we have in the `f-consumer-oidc` package](https://github.com/justeat/fozzie-components/blob/5272c2f44f47b59be3c75e267729f89208db2058/packages/services/f-consumer-oidc/vite.config.js#L11-L21).

You should be able to compare the diff between the resulting bundle before and after making that change to check if the code is excluded.

Reply:
Done and checked filesize.